### ISSUE TITLE
mimic spawning inside the ship

### ISSUE THREAD

**Issue by internet-person** [x753/Lethal-Company-Mimics#1]
so i landed at 41 exp, and there was just a massive fucking exit spawning at the ship.

this did not happen before the latest patch.

**Comment by JakeLoustone** [comment]
Oh, I see what might have happened. It looks like this debug code got pushed out in the 1.0.1 release by accident.

```
                    // Sometimes we need to spawn a mimic near spawn for testing
                    if (mIndex == 0)
                        mimic.transform.position = new Vector3(-7f, 0f, -10f);
```

**Comment by internet-person** [comment]
goddamn that was fast.

**Comment by JakeLoustone** [comment]
This looks to have been fixed in the 1.0.2 release. Many thanks!
